src/ldp/http/SimpleBodyParser.ts

```diff
@@ -41,9 +43,16 @@ export class SimpleBodyParser extends BodyParser {
       contentType: mediaType,
     };
 
+    // Catch parsing errors and emit correct error
+    // Node 10 requires both writableObjectMode and readableObjectMode
+    const errorStream = new PassThrough({ writableObjectMode: true, readableObjectMode: true });
```

Comment:
Isn't it possible to just set `objectMode`?

Reply:
That is what I had initially, but for some reason that caused the tests to fail on Node 10. So must be something they fixed in later versions.

Reply:
Ok, fair enough, was just wondering.